src/AppInstallerCLICore/Commands/FontCommand.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "FontCommand.h"
+#include "Workflows/CompletionFlow.h"
+#include "Workflows/WorkflowBase.h"
+#include "Workflows/FontFlow.h"
+#include "Resources.h"
+
+namespace AppInstaller::CLI
+{
+    using namespace AppInstaller::CLI::Execution;
+    using namespace AppInstaller::CLI::Workflow;
+    using namespace AppInstaller::Utility::literals;
+    using namespace std::string_view_literals;
+
+    Utility::LocIndView s_FontCommand_HelpLink = "https://aka.ms/winget-command-help"_liv;
```

Comment:
```suggestion
    Utility::LocIndView s_FontCommand_HelpLink = "https://aka.ms/winget-command-font"_liv;
```

This should be `font` i presume